service/src/test/resources/core-case-data-store-api/care-order.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Remove this and use CaseDataGeneratorHelper as discussed :)

Reply:
see above comment